torch/csrc/jit/codegen/cuda/kernel_ir_builder.h

```diff
@@ -77,6 +77,8 @@ class TORCH_CUDA_CU_API IrBuilder {
   Bool* falseVal();
   Bool* trueVal();
 
+  NamedScalar* magicZero();
```

Comment:
nit: I'd prefer `magicZeroVal` for consistency with the other shortcuts in the above